config/features.rb

```diff
@@ -28,4 +28,8 @@
   feature :notes_tab_form,
           default: false,
           description: "Enables the Notes tab in the deposit form"
+
+  feature :cache_enabled,
```

Comment:
Can the feature be renamed to something more specific, rather than "cache_enabled". It's a big system and there could be more caches added in the future, so this should be "api_caching"  or similar.